build-system/tasks/build.js

```diff
@@ -112,6 +112,7 @@ build.flags = {
   version_override: '  Overrides the version written to AMP_CONFIG',
   custom_version_mark: '  Set final digit (0-9) on auto-generated version',
   watch: '  Watches for changes in files, re-builds when detected',
+  defineExperimentConstant: '  Build with experiment const set to true',
```

Comment:
All our flags use `snake_case`. Can you rename the flag (here and in all call sites) to `define_experiment_constant`?

Also a nit: For more clarity, change the message to:
> Builds runtime with the EXPERIMENT constant set to true

Reply:
done